src/components/Attachment/AttachmentActions.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import type { Action } from 'stream-chat';
+
+/**
+ * AttachmentActions - The actions you can take on an attachment
+ *
+ * @example ../../docs/AttachmentActions.md
+ */
+export type AttachmentActionsProps = {
+  /** A list of actions */
+  actions: Action[];
+  //   /** Unique id for action button key. Key is generated by concatenating this id with action value - {`${id}-${action.value}`} */
+  id: string;
+  /** The text for the form input */
+  text: string;
+  /** Click event handler */
+  actionHandler?: (
+    event: React.MouseEvent<HTMLButtonElement, MouseEvent>,
+    name?: string,
+    value?: string,
+  ) => void;
+};
+
+const UnMemoizedAttachmentActions: React.FC<AttachmentActionsProps> = (
+  props,
+) => {
+  const { actionHandler, actions, id, text } = props;
+
+  const handleActionClick = (
+    e: React.MouseEvent<HTMLButtonElement, MouseEvent>,
```

Comment:
change all single letter e variables to event, only single letter variable to be used is e for error